beacon-chain/rpc/beacon/attestations.go

```diff
@@ -233,7 +246,64 @@ func (bs *Server) StreamAttestations(
 func (bs *Server) StreamIndexedAttestations(
 	_ *ptypes.Empty, stream ethpb.BeaconChain_StreamIndexedAttestationsServer,
 ) error {
-	return status.Error(codes.Unimplemented, "Unimplemented")
+	attestationsChannel := make(chan *feed.Event, 1)
+	attSub := bs.AttestationNotifier.OperationFeed().Subscribe(attestationsChannel)
+	defer attSub.Unsubscribe()
+	for {
+		select {
+		case event := <-attestationsChannel:
+			if event.Type == operation.UnaggregatedAttReceived {
+				data, ok := event.Data.(operation.UnAggregatedAttReceivedData)
+				if !ok {
+					// Got bad data over the stream.
+					continue
+				}
+				if data.Attestation == nil {
+					// One nil attestation shouldn't stop the stream.
+					continue
+				}
+				epoch := helpers.SlotToEpoch(bs.HeadFetcher.HeadSlot())
+				committeesBySlot, _, err := bs.retrieveCommitteesForEpoch(stream.Context(), epoch)
```

Comment:
Is this cached and/or cheap?

Reply:
there's a cache underneath the hood, it isn't a huge load on slasher but we'll have to do a bit more testing with mainnet/testnet conditions

Reply:
this `epoch` can only be current or previous epoch correct? If yes, it's cheap as they are already cached underneath the hood

Reply:
Yep only current!